Makefile

```diff
@@ -1,22 +1,46 @@
+GOOS ?= `go env GOOS`
+GOPATH ?= `go env GOPATH`
+GOBIN ?= $(GOPATH)/bin
+GO111MODULE = auto
+
 REGISTRY ?= quay.io
 REGISTRY_ACCOUNT ?= kubev2v
 REGISTRY_TAG ?= devel
-CONTROLLER_IMAGE ?= ${REGISTRY}/${REGISTRY_ACCOUNT}/forklift-controller:${REGISTRY_TAG}
-API_IMAGE ?= ${REGISTRY}/${REGISTRY_ACCOUNT}/forklift-api:${REGISTRY_TAG}
-VALIDATION_IMAGE ?= ${REGISTRY}/${REGISTRY_ACCOUNT}/forklift-validation:${REGISTRY_TAG}
-VIRT_V2V_IMAGE ?= ${REGISTRY}/${REGISTRY_ACCOUNT}/forklift-virt-v2v:${REGISTRY_TAG}
-OPERATOR_IMAGE ?= ${REGISTRY}/${REGISTRY_ACCOUNT}/forklift-operator:${REGISTRY_TAG}
-OPERATOR_BUNDLE_IMAGE ?= ${REGISTRY}/${REGISTRY_ACCOUNT}/forklift-operator-bundle:${REGISTRY_TAG}
-OPERATOR_INDEX_IMAGE ?= ${REGISTRY}/${REGISTRY_ACCOUNT}/forklift-operator-index:${REGISTRY_TAG}
-GOOS ?= `go env GOOS`
-GOBIN ?= ${GOPATH}/bin
-GO111MODULE = auto
 
-ifeq (, $(shell which docker))
-    CONTAINER_CMD = podman
-else
-    CONTAINER_CMD = docker
-endif
+VERSION ?= 99.0.0
+NAMESPACE ?= konveyor-forklift
+CHANNELS ?= development
+DEFAULT_CHANNEL ?= development
+
+# Request access to /dev/kvm via Kubevirt's Device Manager
+# That is to ensure the appliance virt-v2v uses would not
+# run in emulation mode, which is significantly slower
+VIRT_V2V_DONT_REQUEST_KVM ?= false
+
+# Use OPM_OPTS="--use-http" when using a non HTTPS registry
+OPM_OPTS ?=
+
+# By default use the controller gen installed by the 
+# 'controller-gen' target 
+DEFAULT_CONTROLLER_GEN ?= $(GOBIN)/controller-gen
```

Comment:
I suggest using `:=` here. I don't think we want users to change the default.

Reply:
Done